### ISSUE TITLE
cloudwatch error in Lambda decoder.

### ISSUE THREAD

**Issue by PatrickMans** [terraform-aws-modules/terraform-aws-notify-slack#22]
With the simple example, I get the following cloudwatch error:

```
Expecting value: line 1 column 1 (char 0): JSONDecodeError
Traceback (most recent call last):
File "/var/task/notify_slack.py", line 76, in lambda_handler
message = json.loads(event['Records'][0]['Sns']['Message'])
File "/var/lang/lib/python3.6/json/__init__.py", line 354, in loads
return _default_decoder.decode(s)
File "/var/lang/lib/python3.6/json/decoder.py", line 339, in decode
obj, end = self.raw_decode(s, idx=_w(s, 0).end())
File "/var/lang/lib/python3.6/json/decoder.py", line 357, in raw_decode
raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

I am trying to send an SNS message via CLI: 

`aws sns publish --topic-arn --topic-arn <my_sns_topic> --message " This is a test" `
upon which I receive a MessageId.

What am I doing wrong here?

**Comment by ryron01** [comment]
This is caused by the SNS event, which is a dictionary, being loaded as json on line 76.
I changed it to 
`message = event['Records'][0]['Sns']['Message']`

**Comment by antonbabenko** [comment]
This issue has been just fixed in [v1.11.0](https://github.com/terraform-aws-modules/terraform-aws-notify-slack/releases/v1.11.0).

Kudos to @ryron01 !
